crate/src/page/partial/content_control_panel.rs

```diff
@@ -73,11 +73,11 @@ fn view_mode_toggle(in_prerendering: bool, mode: Mode) -> Node<Msg> {
                 C.text_gray_500,
                 C.border,
                 C.border_gray_400,
-                C.cursor_pointer,
                 C.rounded_full,
-                C.hover__underline,
-                C.hover__text_gray_700,
-                C.hover__border_gray_600,
+                IF!(!in_prerendering => C.cursor_pointer),
+                IF!(!in_prerendering => C.hover__underline),
+                IF!(!in_prerendering => C.hover__text_gray_700),
+                IF!(!in_prerendering => C.hover__border_gray_600),
```

Comment:
Does it make sense to write it like `IF!(not(in_prerendering) => vec![C.xxx, C.yyy])`?

Reply:
I forgot about this option. I'll fix this.